compiler/one-cmds/one-codegen

```diff
@@ -94,15 +95,46 @@ def _get_parser(backends_list):
     return parser
 
 
-def _verify_arg(parser, args):
+def _verify_arg(parser, args, cfg_args):
     """verify given arguments"""
+    if oneutils.is_valid_attr(cfg_args, 'backend') and oneutils.is_valid_attr(
+            cfg_args, 'backends') and oneutils.is_valid_attr(cfg_args, 'command'):
+        parser.error(
+            '\'backend\' option and \'backends\' option cannot be used simultaneously.')
```

Comment:
I make it throw an error when the user uses 'backends' and 'backend' key simultaneously in cfg file for simplicity. Otherwise, the logic would be too complicated.. (maybe already it is complicated 😢)

Reply:
Reducing the number of cases looks good to me.